src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -521,6 +550,31 @@ def makeTransferRucio(self, wflow, dataIn, dids, dataSize, grouping, copies, nod
             self.logger.info(msg, wflow.getName(), dids, rseExpr, ruleAttrs)
         return success, transferId
 
+    def getRids(self, workflowName):
```

Comment:
Please be more explicit with this method name. Perhaps something like "getRuleIdsFromDoc" etc

Reply:
ok, renamed.